packages/magento-cart-items/CartItems/CartItems.tsx

```diff
@@ -1,24 +1,41 @@
-import { RenderType, TypeRenderer } from '@graphcommerce/next-ui'
-import { Box } from '@mui/material'
+import { ActionCardLayout, ActionCardLayoutProps, nonNullable } from '@graphcommerce/next-ui'
 import { CartItemsFragment } from '../Api/CartItems.gql'
+import { CartItemActionCard, CartItemActionCardProps } from '../CartItem/CartItemActionCard'
 
-export type CartItemRenderer = TypeRenderer<NonNullable<NonNullable<CartItemsFragment['items']>[0]>>
-
-export type CartProps = { renderer: CartItemRenderer } & CartItemsFragment
+export type CartProps = ActionCardLayoutProps & {
+  cart?: CartItemsFragment | null
+  itemProps?: Omit<
+    CartItemActionCardProps,
+    'cartItem' | 'layout' | 'onClick' | 'disabled' | 'selected' | 'reset' | 'color'
+  >
+} & { size?: 'small' | 'medium' | 'large' }
 
 export function CartItems(props: CartProps) {
```

Comment:
Can we perhaps create a completely new component CartItemsCards or something, so we can keep the old behavior for now? The old component can be marked as `@deprecated`.